collectors/pool_usage.go

```diff
@@ -97,6 +115,7 @@ type cephPoolStats struct {
 		ID    int    `json:"id"`
 		Stats struct {
 			BytesUsed json.Number `json:"bytes_used"`
+			MaxAvail  float64     `json:"max_avail"`
```

Comment:
Would it be possible to move rest of the fields over to `float64` too? I'd like to have this be consistent as much as possible. Thanks! :)